`npm test` fails with timeout

To reproduce, run the following commands:
- `git clone https://github.com/strongloop/loopback4-example-shopping`
- `cd loopback4-example-shopping`
- `npm install`
- `npm test`

It fails with:
```
> lb-mocha --allow-console-logs "packages/*/dist/__tests__/**/*.js"

  !

0 passing (15ms)
1 failing
1) "before all" hook in "{root}":
Error: connect ENOENT //./pipe/docker_engine
at PipeConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)
npm ERR! Test failed.  See above for more details.
```

@raymondfeng Hi Raymond, I'm getting the same error below on Win 10 and Ubuntu 19.  Do you have any ideas?

````node
> loopback4-example-shopping-monorepo@1.1.1 test /home/user/dev/loopback4-example-shopping
> lb-mocha --allow-console-logs "packages/*/dist/__tests__/**/*.js"



  !

  0 passing (33s)
  1 failing

  1) "before all" hook in "{root}":
     Error: Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
      at listOnTimeout (internal/timers.js:531:17)
      at processTimers (internal/timers.js:475:7)



npm ERR! Test failed.  See above for more details.
````

> It requires a docker engine to be running on the Windows machine.

Doesn't work with Docker running

```
> lb-mocha --allow-console-logs "packages/*/dist/__tests__/**/*.js"                                                                                                                                                                                                                                                                                                                                             

  !                                                                                                                                                                                                                                                                                                                                                                                                               0 passing (30s)                                                                                                                                                                                         1 failing                                                                                                                                                                                                                                                                                                                                                                                                       1) "before all" hook in "{root}":                                                                                                                                                                          Error: Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.                                                                    at listOnTimeout (internal/timers.js:531:17)                                                                                                                                                            at processTimers (internal/timers.js:475:7)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
npm ERR! Test failed.  See above for more details.                                              
```

--- a/packages/shopping/src/__tests__/fixtures/redis-mongo-docker.ts
+++ b/packages/shopping/src/__tests__/fixtures/redis-mongo-docker.ts
@@ -7,7 +7,7 @@ import {GenericContainer, StartedTestContainer} from 'testcontainers';
 
 async function startRedis() {
   const container = await new GenericContainer('redis')
-    .withName('redis_lb4_shopping')
+    .withName('redis_lb4_shopping_test')
     .withExposedPorts(6379)
     .start();
   process.env.SHOPPING_APP_REDIS_MASTER_SERVICE_HOST = container.getContainerIpAddress();
@@ -19,7 +19,7 @@ async function startRedis() {
 
 async function startMongoDB() {
   const container = await new GenericContainer('mongo')
-    .withName('mongodb_lb4_shopping')
+    .withName('mongodb_lb4_shopping_test')
     .withExposedPorts(27017)
     .start();
   process.env.SHOPPING_APP_MONGODB_SERVICE_HOST = container.getContainerIpAddress();

